docs/docs-content/clusters/edge/trusted-boot/keys/key-management.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_label: "Key Management"
+title: "Trusted Boot Key Management"
+description: "Learn about how to manage the keys used in Trusted Boot."
+hide_table_of_contents: false
+sidebar_position: 15
+tags: ["edge"]
+---
+
+Several key pairs are used in Trusted Boot during installer ISO generation, upgrade image generation, as well as
+installation. Each pair of keys serves a different purpose, is used during different stages of Edge artifact building
+and deployment, and needs to be secured differently. This page discusses the different key pairs used by Trusted Boot
+and how to secure them.
+
+Careful key management is the foundation of all security benefits provided by Trusted Boot. All security provided by
+Trusted Boot assumes that your keys are handled and stored securely. Ensure that you follow our recommendations to avoid
+compromising the security of your systems.
+
+## Platform Key (PK)
+
+The private PK must be stowed away in a secure location **immediately** after being generated. You do not need the PK
+private key during EdgeForge operations, installation, upgrades or deployments of your Edge hosts. The public PK key is
+required during the EdgeForge build process so that it can be embedded into the Edge Installer ISO and thereafter
+installed on Edge hosts.
+
+:::danger
```

Comment:
not sure where to place this comment. but another **danger** item is if people re-run genkey if they used self-signed, it will overwrite everything (silently). people should be very careful about ensuring they have backed up/stored securely all of the important keys.